src/components/DefaultModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from 'styled-components';
+import MainButton, { Variant } from './MainActionButton';
+
+export const Overlay = styled.div`
+	position: absolute;
+	backdrop-filter: blur(5px);
+	height: 100%;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+`;
+
+export const ButtonField = styled.div`
+	margin-right: 1em;
+	margin-left: 10.5em;
```

Comment:
I don't think we need both of these... You make just need to do 

`justify-content: flex-end` or `align-items: flex end` then add whatever margin/padding you feel on the right side (1em seems good)

Those flex-end properties mean to put everything to one side (in this case, because it is a row, it will be to the right I believe)

Reply:
I believe I tried this but it wasn't wanting to work due to `width: 100%;` being set on `MainActionButton`